Packages/com.unity.multiplayer.samples.coop/CHANGELOG.md

```diff
@@ -1,5 +1,110 @@
+* Fixed the ignored health amount (HP parameter) for revived characters. The correct value correctly sets the revived character to a lower amount than maximum. 
+
+### Fixes
+
+This release includes the following issue fixes:
+
+* Fixed an issue where any player that plays the game and then returns to the Main Menu may be unable to Start or Join properly again, requiring you to restart the client.
+* Updates and refinements in Boss Room resolved an issue that occurred in some degraded network conditions that caused a replicated entity on a client to vanish from that client, creating the effect of being assailed by an invisible enemy. 
+* Fixed displayed graphical affects for casting and blocking a Bolt to correctly match the caster and target, and properly stops animations for cancelled actions across clients. 
+* Fixed a rare exception when the Tank character uses her Shield Aura ability to intercept the Boss charge attack. 
+* Fixed an issue returning all clients to the Main Menu when the host leaves the Boss Room. 
+* Green quads no longer show on impact when the Archer arrow strikes enemies. 
+* Fixed issue to correctly allow one player to receive a character when two players in the Character Select click **Ready** for the same hero at the same time. Character Select is no longer blocked.  
+* Fixed an issue with boss collisions with a Pillar correctly applying a stun effect and shattering the pillar when using the Trample attack. 
+* Fixed the lobby welcome messages to correctly list the player names, including a previous issues for P1 and P2. 
+* On Windows, investigated and fixed issues with visible effects for character actions including Mage Freeze attack and Rogue Dash.
+* On Wizards, fixed issue with imp spawners not respawning new imps after exploring the room.
+* Fixed an issue where the door state does not reflect the existing state when players connect late to a game, for example if other players open the door and a player joins late the door displays as closed. 
+* Removed a previous work-around for character selections when host replays a completed game. The issue was resolved, allowing players to see character selections during replay. 
+* Fixed collision wall settings, fixing an issues where the boss knock-back ability sent players through walls.
+* Resolved an issue where any players leaving the lobby sent all players to the lobby.
+* Fixed animations for enemies including the smoke animation for destroyed imps and the boss helmet when crying.
+
+### Known issues
+
+* An MLAPI soft sync error on cleanup between scene transitions may break the game, for example imps do not spawn and pots are intangible.
+* Sometimes after completing a match and the host starts a new match from the Victory or Loss screen, connected players may have no visible interactions to join or select characters. A work-around is implemented to not block entry into the game. 
+
```

Comment:
This was the other known issue in question: https://unity3d.atlassian.net/browse/GOMPS-534.

Reply:
Adding!